src/assets/scripts/templates/product.js.liquid

```diff
@@ -52,11 +52,27 @@ ira.cacheSelectors = function () {
   };
 };
 
+var bvEnableScrollToReviews = function () {
+  if ($('.bv_button_buttonFull').length ) {
+      $('.bv_button_buttonFull').on('click', function () {
+        $('#reviewsTab').find('a').trigger('click');
+        $('html, body').animate({
+            scrollTop: $("#reviewsTab").offset().top
+        }, 500);
+      });
+  } else {
+    setTimeout(function () {
```

Comment:
I'm really not a huge fan of looping like this. Can we wrap whatever element houses the bv_button_buttonFull with a href that links to an anchor tag? Setting the active tab will probably take a touch of js. I don't like this as if bazaarvoice goes down we'll have a script running every 50ms until it's resolved.